gophish-main.tf

```diff
@@ -0,0 +1,112 @@
+provider "aws" {
+  region     = "eu-central-1"
+}
+
+// details of the aws instance
+resource "aws_instance" "example" {
+  ami = "ami-40d5672f"
+  instance_type = "t2.micro"
+  vpc_security_group_ids = ["${aws_security_group.instance.id}"]
+  key_name = "${aws_key_pair.auth.id}"
+
+  tags {
+    Name = "phishing-machine"
+  }
+
+  user_data = <<HEREDOC
+    #!/bin/bash
+    yum update -y
+    yum install wget -y
+    yum install unzip -y
+    su ec2-user
+    cd /home/ec2-user/
+    wget https://github.com/gophish/gophish/releases/download/v0.6.0/gophish-v0.6.0-linux-64bit.zip
```

Comment:
I've set up some Cloudflare magic to make it easier to get the latest version if you're interested. The format is `getgophish.com/releases/latest/<os>/<arch>`. So, we could change this to something like:

`wget https://getgophish.com/releases/latest/linux/64 -O gophish-linux-64bit.zip` and then update the call to `unzip` further down to take into account this new ZIP filename.

Reply:
Updated this, thank you :)